src/js/jsx/shared/Datalist.jsx

```diff
@@ -671,6 +671,7 @@ define(function (require, exports, module) {
 
             var dropDownClasses = classnames({
                 "drop-down": true,
+                "control-group__vertical": true,
```

Comment:
Remove this (we may not want EVERY datalist to have this class). Instead, you can wrap that component in a div that has that class